views/pages/fullEpisodes.ejs

```diff
@@ -0,0 +1,17 @@
+
+<ul>
+    <li><a href="/">Home</a></li>
+    <li><a href="/fullEpisodes">Full Episodes</a></li>
+    <li><a href="/Contact">Contact Us</a></li>
+</ul>
+
+
+<% for (let episode of episodeArray) { %>
+    <div class="card">
+        <a href="/singleEpisode/<%= episode.id%>">
+            <h3>Title: <%= episode.name %> <%= makeEpisodeCode(episode)%></h3>
+        </a>
+        <img src="<%= episode.image.medium %>" alt="Episode Cover Image"/>
```

Comment:
Ideally we wouldn't set the alt text to "Episode cover image" but to a description of the image, or at least something interesting for the site audience.  
Imagine using a screenreader and listening to the description of the images as they occur in the flow of the page: "Heading: Winter is coming.   An image is shown of an 'Episode Cover Image'".

Still, we don't have that image description in the data.  you might keep the alt text brief, then, if it can't be helpful.